developing-with-nats/tutorials/jwt.md

```diff
@@ -0,0 +1,1540 @@
+This shows how to take an identity key offline.
+Identity NKEY of the operator/account is the only one allowed to modify the corresponding JWT and thus add/remove signing keys.
+Thus, initial signing keys are best created and assigned prior to removing the private identity NKEY.
+
+Basic strategy: take them offline & delete in [`nsc`](#nsc) NKEY directory
+
+Use `nsc env` to determine your NKEY directory. (Assuming `~/.nkeys` for this example)
+`nsc list keys --all` lists all keys under your operator and indicates if they are present and if they are signing keys.
+
+Keys for your Operator/Account can be found under `<nkyesdir>/keys/O/../<public-nkey>.nk` or `<nkyesdir>/keys/A/../<public-nkey>.nk`
+The operator identity NKEY ODMFND7EIJ2MBHNPO2JHCKOZIAY6NAK7OT4V2ZT2C5O6LEB3DPKYV3QL would reside under `~/.nkeys/keys/O/DM/ODMFND7EIJ2MBHNPO2JHCKOZIAY6NAK7OT4V2ZT2C5O6LEB3DPKYV3QL.nk`
+
+*Please note that key storage is sharded by the 2nd and 3rd letter in the key*
+
+Once these files are backed up and deleted `nsc list keys --all` will show them as not stored.
+You can continue as normal, `nsc` will pick up signing keys instead.
+
+Since you typically distribute user keys or creds files to your applications, there is no need for `nsc` to hold on to them in the first place.
+Credentials files are a concatenated user JWT and the corresponding private key, so don't forget to delete that as well.
+
+Key and creds can be found under `<nkyesdir>/keys/U/../<public-nkey>.nk` and `<nkyesdir>/creds/<operator-name>/<account-name>/<user-name>.creds`
+
+#### Reissue Identity NKEYs
+
+If you can easily re-deploy all necessary keys and jwt, simply re-generating a new account/user (possibly operator) this will be the simplest solution.
+The steps necessary are identical to the initial setup, which is why it would be preferred.
+In fact, for user NKEYs and JWT, generating and distributing now ones to affected applications it is the best option.
+
+Even if regeneration of an account or operator is not your first choice, it may be your method of last resort. 
+Below sections outline the steps this would entail.
+
+##### Operator
+
+In order to reissue an operator identity NKEY use `nsc reissue operator`.
+It will generate a new identity NKEY and use it to sign the operator.
+`nsc` will also resign all accounts signed by the original identity NKEY. 
+Accounts signed by operator signing keys will remain untouched.
+
+The altered operator JWT will have to be deployed to all affected `nats-server` (one server at a time).
+Once all `nats-server` have been restarted with the new operator, push the altered accounts.
+Depending on your deployment mode you may have to distribute the operator JWT and altered account JWT to all other [`nsc`](#nsc) environments.
+
+This process will be a lot easier when operator signing keys where used throughout and no account will be re signed because of this.
```

Comment:
```suggestion
This process will be a lot easier when operator signing keys where used throughout and no account will be re-signed because of this.
```